src/main/java/de/rub/nds/modifiablevariable/Logger/CustomLayout.java

```diff
@@ -0,0 +1,562 @@
+/*
+ * ModifiableVariable - A Variable Concept for Runtime Modifications
+ *
+ * Copyright 2014-2022 Ruhr University Bochum, Paderborn University, and Hackmanit GmbH
+ *
+ * Licensed under Apache License 2.0 http://www.apache.org/licenses/LICENSE-2.0
+ */
+package de.rub.nds.modifiablevariable.Logger;
```

Comment:
The package should be `de.rub.nds.modifiablevariable.logging` or similar (notice the lowercase package name here as per Java coding conventions).